clmm/polaraveraging.py

```diff
@@ -159,14 +162,14 @@ def compute_tangential_and_cross_components(cluster=None,
 
     # Compute the lensing angles
     if geometry == 'flat':
-        angsep, phi = _compute_lensing_angles_flatsky(ra_lens, dec_lens, ra_source_list,
-                                                      dec_source_list)
+        angsep, phi = _compute_lensing_angles_flatsky(ra_lens, dec_lens, np.array(ra_source_list),
```

Comment:
The docs mention ra_source_list as "array like".  Minor comment/recommendation:  recommend - either specify numpy array as type, thus avoiding the need to wrap as a numpy array, and/or rename "ra{,dec}_source_list" -> "ra{,dec}_sources" because the "_list" implies list type.

Reply:
I removed the misleading `_list` from the variable names and explicitly mention that these should be arrays and not array-like.